docs/development/tech/index.md

```diff
@@ -194,7 +194,7 @@ Always keep in mind:
 - It's easier (and faster) to request a code review on small changes
   (less than 100 lines) than on a huge feature (over 1000
   lines). **Push early, and push often**. If you anticipate a big
-  feature which you want to have code reviewed, start requesting code
+  feature whose code you want to have reviewed, start requesting code
```

Comment:
Haha, I spent some time googling this because I though `whose` would not be correct since it should refer to a person but nope, it's ok as well for inanimate objects since there isn't another word in English: http://www.quickanddirtytips.com/education/grammar/whose-for-inanimate-objects